hazelcast-jet-core/src/main/java/com/hazelcast/jet/Job.java

```diff
@@ -82,6 +83,12 @@ default String getName() {
     @Nonnull
     JobStatus getStatus();
 
+    /**
+     * Returns a snapshot of the current values of all job-specific metrics.
```

Comment:
javadoc needs to be expanded. What does it return for completed jobs?

Reply:
Ok, done in next push.